### PR TITLE
Add indexquickview extension

### PR DESCRIPTION
**Pull request by ErlichLiu**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
IS this ready for another review?

**Comment by ErlichLiu** [comment]
Hi，

You can review now , and I still want to keep the chinese translation, so
I  add the English words.

Is that work for you?

Thank you!

Erlich

Per Nielsen Tikær ***@***.***> 于2023年10月30日周一 17:08写道：

**Comment by pernielsentikaer** [comment]
At the moment, it is not allowed to have any language other than English. Therefore, if you want it in the store, you will need to remove Chinese 😞

**Comment by ErlichLiu** [comment]
Hi,

I have already remove Chinese words.

please review it again.

Thank you

Erlich

Per Nielsen Tikær ***@***.***> 于2023年10月31日周二 17:44写道：

File: extensions/indexquickview/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+          </ActionPanel>
+        }
+      />
+      <List.Item title="SSE 上证指数" subtitle={sh001 === "" ? "Loading..." : sh001} />
+      <List.Item title="SZSE 深证指数" subtitle={hs300 === "" ? "Loading..." : hs300} />
+      <List.Item title="NASDAQ 纳斯达克指数" subtitle={int_nasdaq === "" ? "Loading..." : int_nasdaq} />
+      <List.Item title="S&P 500 标普500指数" subtitle={int_sp500 === "" ? "Loading..." : int_sp500} />
+      <List.Item title="DJIA 道琼斯指数" subtitle={int_dji === "" ? "Loading..." : int_dji} />
+      <List.Item title="STI 新加坡指数" subtitle={b_FSSTI === "" ? "Loading..." : b_FSSTI} />
+      <List.Item title="Nikkei 225 日经指数" subtitle={int_nikkei === "" ? "Loading..." : int_nikkei} />
```
Comment (pernielsentikaer) [inline extensions/indexquickview/src/index.tsx:204]: This looks like chinese to me 🙂

**Comment by ErlichLiu** [comment]
Hi,

Sorry , this is my fault. I forgot run git push 😂

It's ok now.

Thanks

Erlich

Per Nielsen Tikær ***@***.***> 于2023年11月2日周四 19:50写道：

**Comment by pernielsentikaer** [comment]
There is still Chinese 😊

**Pull request closed without merging**